ios/ZulipMobile/Assets.xcassets/LaunchScreen.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "blank.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "blank-1.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "blank-2.png",
+      "idiom" : "universal",
+      "scale" : "3x"
```

Comment:
Why three different files? It appears that they're all identical (same sequence of bytes.)

Also I read [those Expo instructions](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#-add-an-image-to-imagesxcassets) as not really saying "you need to use an image"; rather they're assuming that the thing you want is an image, and they're providing instructions for how to do so. In particular the reference to "*the* image file that [is desired to] serve as a splash screen" sounds to me like it's assuming there is such an image, meaning an image you want to serve as a splash screen. So I expect you can simplify this with something like the suggestion you make in the commit message: an empty view with a background color, instead of an image view.

Reply:
Makes sense—it works perfectly fine without an image.